package.json

```diff
@@ -40,6 +40,7 @@
     "babel": "^5.8.25",
     "babel-core": "^5.8.25",
     "babel-eslint": "^4.1.1",
+    "babel-relay-plugin": "0.7.1",
```

Comment:
I just updated the path here: https://github.com/facebook/relay/blob/master/scripts/jest/preprocessor.js#L67 to the one in `scripts` and that worked for me.